src/test/java/org/teamresistance/frc/subsystem/drive/DriveTest.java

```diff
@@ -0,0 +1,63 @@
+package org.teamresistance.frc.subsystem.drive;
+
+import org.junit.jupiter.api.Test;
+import org.strongback.components.Motor;
+import org.strongback.drive.MecanumDrive;
+import org.strongback.mock.Mock;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class DriveTest {
+  // Mock motors allow us to verify the motor outputs are correct without needing the actual
+  // hardware. The speed of the motor is stored in memory for later retrieval by getSpeed().
+  private final Motor leftFront = Mock.stoppedMotor();
+  private final Motor leftRear = Mock.stoppedMotor();
+  private final Motor rightFront = Mock.stoppedMotor();
+  private final Motor rightRear = Mock.stoppedMotor();
+
+  // We're using a real MecanumDrive instance instead of a mock because the logic inside
+  // MecanumDrive is part of the system we're testing. It's in an instance field rather than
+  // being inlined to allow us to directly call robotDrive.cartesian()/polar() later and compare
+  // the results to calling drive.onSignal().
+  private final MecanumDrive robotDrive = new MecanumDrive(leftFront, leftRear, rightFront,
+      rightRear, Mock.angleSensor());
+
+  // Drive is system-under-test (SUT), which includes the MecanumDrive dependency. We can treat
+  // it as an isolated black box where its methods are the inputs and the motors are the outputs.
+  private Drive drive = new Drive(robotDrive, () -> 0, () -> 0, () -> 0);
+
+  @Test
+  void onSignal_WithRobotOrientedSignal_ShouldDrivePolar() {
```

Comment:
This is fine if that's the convention we want to go with. but it's kind of clunky. Maybe we don't need the `onSignal_` part of the name? Other than that, this test is good.  I would also add a method description and say something like "testing to see if the robot oriented drive system gives the same outputs as the library's robot oriented drive system". I know you mentioned this elsewhere, but it's nice to have it as a method description because that's where you first look.